### ISSUE TITLE
Tag Picker: Keros error if no element is selected

### ISSUE THREAD

**Issue by neeraj-khanna** [OfficeDev/office-ui-fabric-react#6656]
![tagpickerbug](https://user-images.githubusercontent.com/14049149/46786789-8f013200-cd53-11e8-9355-b6209938360e.PNG)

### Bug Report 
 
- __Package version(s)__: 6.83.0
- __Browser and OS versions__: Chrome Version 69.0.3497.100 (Official Build) (64-bit)
 
#### Describe the issue:
 
If no element is selected in Tag Picker control, Keros gives following error:

*Certain ARIA roles must contain particular children*
 
#### Actual behavior:
 
Keros is giving error that given Aria role needs children.
 
#### Expected behavior:
 
No Keros issue.

**Comment by micahgodbolt** [comment]
hmm...seems a list needs to have list items. Does that mean that it's not a list until you select at least 1 thing?

**Comment by kenotron** [comment]
I cannot repro this any longer, there is a single input tag with role "combobox". Then when you pick one item, it would create a new list with a listitem. Indeed latest AI check on the page does not repro this problem any longer.
